webpack.config.js

```diff
@@ -17,16 +17,11 @@ var babelPlugin = new webpack.DefinePlugin({
 });
 
 var devPlugins = [
-  babelPlugin
+  babelPlugin,
 ];
 
 var prodPlugins = [
   babelPlugin,
-  new webpack.DefinePlugin({
-    optimization: {
-      minimizer: [new UglifyJsPlugin()]
```

Comment:
Why was this removed? Don't we want minified code in prod?

Reply:
So actually using webpacks `mode` config when marked as production does all this for us, turned out this was just redundant: https://webpack.js.org/concepts/mode/#mode-production